include/gc/ExecutionEngine/CPURuntime/ConstantCache.hpp

```diff
@@ -0,0 +1,137 @@
+#ifndef GC_EXECUTIONENGINE_CPURUNTIME_CONSTANT_CACHE_H
```

Comment:
License header here.

Reply:
Sure. This PR is WIP for preview, since it is has pending dependency. I will update the code when it is finalized.

Reply:
@Menooker 
Please rebase `yijie/pipeline` and `yijie/mainfunc_wrapper` to enable the license check workflow